pkg/keyspace/keyspace.go

```diff
@@ -111,7 +111,10 @@ type CreateKeyspaceRequest struct {
 type CreateKeyspaceByIDRequest struct {
 	// ID of the keyspace to be created.
 	// Using an existing ID will result in error.
-	ID     *uint32
+	ID *uint32
+	// Name of the keyspace to be created.
+	// Using an existing name will result in error.
+	Name   string
```

Comment:
Do we need to add a test for existed name?

Reply:
done